Presentation.Web/Tests/Helpers/KendoLoaderHelper.ts

```diff
@@ -10,7 +10,7 @@ class KendoLoaderHelper {
 
     waitForKendoGridData(columnName: protractor.ElementFinder) {
         console.log("Waiting for kendo grid to load");
-        return browser.wait(this.kendoLoadingMaskOff, this.waitUpTo.twentySeconds)
+        return browser.wait(this.kendoLoadingMaskOff, this.waitUpTo.fourtySeconds)
```

Comment:
Jeg kan umiddelbart ikke se noget der skulle være galt her udover at serveren har været for langsom til at levere overblikket fordi den har været lidt presset. Så prøver at give den lidt mere tid her.